tests/Dockerfile.fedora

```diff
@@ -61,7 +63,10 @@ RUN rm -f /usr/lib/rpm/macros.d/macros.transaction_ima
 RUN dnf -y --enablerepo=updates install "sequoia-sq >= 1.0"
 RUN dnf clean all
 
-RUN echo "%_dbpath $(rpm --eval '%_dbpath')" > /root/.rpmmacros
+ARG PREFIX
+# Make sure the built rpm find the system db (for shell usage)
+RUN mkdir -p ${PREFIX}/etc/rpm
+RUN echo "%_dbpath $(rpm --eval '%_dbpath')" > ${PREFIX}/etc/rpm/macros.db
```

Comment:
This is still within the "base" layer which is combined with the native cmake build of rpm. So the prefix depends on that cmake build. IOW, that's why `~/.rpmmacros` was used originally :laughing: 

We would have to pass `--build-arg PREFIX=...` when building the "base" layer in mktree.oci but... that'd just make things more complicated.

I'd say let's just remove drop the macros.db file altogether for now and revisit later. Perhaps even by changing the dbpath in our own macros file.

Reply:
Aargh :facepalm:

Reply:
But okay, dropping the _dbpath thing is fine by me. And that means we don't actually need any of these Dockerfile changes. Configurable prefix has potential use-cases but its not interesting to me just now.

Reply:
Yeah, I was just thinking the same - once we forget the db path thing, this whole ARG ordeal becomes... meaningless :sweat_smile: